randovania/data/json_data/prime2/Temple Grounds.txt

```diff
@@ -209,10 +209,11 @@ Asset id: 2606692981
       Any of the following:
           Morph Ball and Boost Ball
           All of the following:
-              Space Jump Boots and Standable Terrain (Intermediate)
+              Space Jump Boots
               Any of the following:
-                  Before Torvus Energy Returned and Slope Jump (Intermediate)
-                  Morph Ball and Morph Ball Bomb and Bomb Space Jump (Intermediate)
+                  Before Torvus Energy Returned and Slope Jump (Intermediate) and Standable Terrain (Intermediate)
+                  Morph Ball and Morph Ball Bomb and Bomb Space Jump (Intermediate) and Standable Terrain (Intermediate)
```

Comment:
I'd combine this under standable terrain for simplicity

Reply:
done